packages/ts/form/src/BinderNode.ts

```diff
@@ -338,8 +338,8 @@ export class BinderNode<T, M extends AbstractModel<T>> {
       // from initial `binder.read()` or `binder.clear()` or by using a
       // binder node (e. g., form biding) for a nested field.
```

Comment:
A good chance to fix the comment ;)

```suggestion
      // binder node (e. g., form binding) for a nested field.
```